specs-jetpack-calypso/wp-jetpack-connect-spec.js

```diff
@@ -50,6 +50,11 @@ test.before( function() {
 test.describe( `Jetpack Connect: (${ screenSize })`, function() {
 	this.timeout( mochaTimeOut );
 
+	test.after( function() {
+		const jnFlow = new JetpackConnectFlow( driver, 'jetpackConnectUser' );
```

Comment:
I think we just need to use `jetpackConnectUser` in `Disconnect expired sites` block. Looks like that username parameter was lost in one of refactors.

Reply:
Yeah, I was confused by that magic too :)

Thanks for the update!